### PR TITLE
Add RoBERTa Mojo example

### PR DESCRIPTION
**Pull request by tzhenghao**
This PR introduces an example of how to run RoBERTa with Mojo

## Tests

Screenshot 2024-02-29 at 3 12 30 PM

### CODE REVIEW

File: examples/inference/roberta-mojo-tensorflow/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    [model card](https://huggingface.co/microsoft/RoBERTa).
+
+- `simple-inference.py`
+    Classifies example input statement using the MAX Engine. The script prepares an
+    example input, executes the model, and generates the resultant classification
+    output.
+
+    You can use the `--input` CLI flag to specify an input example.
+    For example, `python3 simple-inference.py --input=<YOUR_INPUT_STRING_HERE>`.
+
```
Comment (alexandrnikitin) [inline examples/inference/roberta-mojo-tensorflow/README.md:36]:
```suggestion
- `simple-inference.🔥`
    Classifies example input statement using the MAX Engine. The script prepares an
    example input, executes the model, and generates the resultant classification
    output.

    You can use the `--input` CLI flag to specify an input example.
    For example, `mojo simple-inference.🔥 --input=<YOUR_INPUT_STRING_HERE>`.

```

File: examples/inference/roberta-mojo-tensorflow/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+setuptools>=69.1.0
+tensorflow>=2.13.0
+transformers>=4.0.0
+tritonclient[http]>=2.42.0
```
Comment (alexandrnikitin) [inline examples/inference/roberta-mojo-tensorflow/requirements.txt:4]:
```suggestion
```

File: examples/inference/roberta-python-tensorflow/run.sh
Comment (alexandrnikitin) [inline examples/inference/roberta-python-tensorflow/run.sh:1]: accidental change? let's make it in a separate PR

**Review by alexandrnikitin — APPROVED** [review]

File: examples/inference/roberta-mojo-tensorflow/README.md (hunk 1 above)
Comment (tzhenghao) [reply examples/inference/roberta-mojo-tensorflow/README.md:36]: Good catch, forgot the `README.md`. Will address

File: examples/inference/roberta-python-tensorflow/run.sh
Comment (alexandrnikitin) [reply examples/inference/roberta-python-tensorflow/run.sh:1]: ok, makes sense then
